functests/performance/hugepages.go

```diff
@@ -0,0 +1,47 @@
+package performance
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+	testclient "github.com/openshift-kni/performance-addon-operators/functests/utils/client"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/nodes"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	pathHugepages1048576kBNumaNode0 = "/sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages"
+	pathHugepages2048kB             = "/sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages"
+)
+
+var _ = Describe("[performance]Hugepages", func() {
+	var workerRTNode *corev1.Node
+
+	BeforeEach(func() {
+		var err error
+		workerRTNodes, err := nodes.GetByRole(testclient.Client, testutils.RoleWorkerRT)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(workerRTNodes).ToNot(BeEmpty())
+		workerRTNode = &workerRTNodes[0]
+	})
+
+	Context("when NUMA node specified", func() {
+		It("should be allocated on the specifed NUMA node ", func() {
+			command := []string{"cat", pathHugepages1048576kBNumaNode0}
+			nrHugepages, err := nodes.ExecCommandOnMachineConfigDaemon(testclient.Client, workerRTNode, command)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(string(nrHugepages)).To(Equal("1"))
```

Comment:
is there anyway to verify the system picked up this change that goes beyond checking the file?

like maybe inspecting ```cat /proc/meminfo  | grep Huge``` ?

also, can we test that the systemd unit didn't fail and return non-zero exit code?

Reply:
`meminfo` shows only information regarding default hugepages, in our case, it will be 1GB, and IIRC it looks exactly at the same file that used under the test

Reply:
cat /sys/devices/system/node/node*/meminfo | grep Huge

Reply:
@MarSik but does it different from `/sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages`?

Reply:
@MarSik and again meminfo just show information regarding default hugepages size